apps/astarte_realm_management/lib/astarte_realm_management/devices/devices.ex

```diff
@@ -26,6 +26,10 @@ defmodule Astarte.RealmManagement.Devices do
     with {:ok, decoded_id} <- Device.decode_device_id(device_id),
          :ok <- ensure_device_exists(realm_name, decoded_id) do
       Queries.insert_device_into_deletion_in_progress(realm_name, decoded_id)
+
+      server_via_tuple()
```

Comment:
Which `server`? Let's help our future selves:

```suggestion
      data_updater_plant_rpc()
```